_tutorials/new-component/create-a-new-component-part-1.md

```diff
@@ -0,0 +1,237 @@
+
+Although Origami components are authored using Bower, components are published to NPM so projects which use Origami may choose to use NPM over Bower ([but we still recommended Bower for now](/docs/tutorials/npm/)).
+
+Don't worry if this sounds a little confusing at the moment. Later in this tutorial we will install dependencies using Bower for our component to use. We will also see how Origami components are automatically published to NPM.
+
+## Start Developing
+
+Now we have a basic component to work from we can start developing!
+
+First we need to install any component dependencies by running the Origami Build Tools `install` command:
+
+<pre><code class="o-syntax-highlight--bash">obt install</code></pre>
+
+Then we can use the `develop` (`dev`) command to start working on the component. The `dev` command creates a server for us to preview our component in the browser. And whenever we make any change to the source code the component will be rebuilt, which we will be able to see by refreshing our browser:
+
+<pre><code class="o-syntax-highlight--bash">obt dev</code></pre>
+
+Opening the link output by the develop command, for example `localhost:8999`, shows us the component demos (html files) and demo assets (built js and css files) in the browser.
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/example-dev-output.png" />
+	<figcaption class="o-typography-caption">
+        Running the dev command builds our demos. We'll discuss demos more shortly.
+	</figcaption>
+</figure>
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/example-demos-local.png" />
+	<figcaption class="o-typography-caption">
+        Opening the link output by the develop command, `localhost:8999` in this case, shows us the component demos (html files) and demo assets (js and css files) in the browser.
+	</figcaption>
+</figure>
+
+Clicking `demo.html` to open that demo will show a blank page. In the next section we will update this demo with markup and content for our component.
+
+## Markup
+
+So Origami components can be used with any backend language or framework users of Origami components copy and paste HTML from the readme `README.md` or demos in the [Origami registry](https://registry.origami.ft.com/components/) (although templates may be provided in some cases [according to the markup section of the component specification](/spec/v1/markup/#templates)). To update the markup in our component demos see the `demos` directory.
+
+The templates for demos are written in [mustache](https://mustache.github.io/), and demos may include their own styles ([Sass](https://sass-lang.com/)) and JavaScript which is not part of the component itself. Note that demo code is not used by projects which depend on the component.
```

Comment:
I don't think we've explained what demos are yet, should we explain what they are, why they are useful etc? This may be the first time a developer is making any type of components whatsoever so they may not know that components usually have live examples of how they look/work.

Reply:
Good point. I don't think we should go into great depth at this point but should expect  a little in the previous section. It's not the first mention of demos:

>Opening the link output by the develop command, for example `localhost:8999`, shows us the component demos (html files) and demo assets (built js and css files) in the browser.